### PR TITLE
IgnoreIP/IgnoreCIDR and internal rules

### PR DESCRIPTION
**Pull request by lubomudr**
Hi
IgnoreIP/IgnoreCIDR does not work for rules without SpecialScores (all internal rules except injections) :worried: 

Additionally, some cosmetic improvements to define internal rules

### CODE REVIEW

File: unit-tests/tests/33ignoreip.t [hunk 1]
```diff
@@ -250,3 +250,26 @@ GET /foobar?a=update/table
...
+     DeniedUrl "/RequestDenied";
+     root $TEST_NGINX_SERVROOT/html/;
+     index index.html index.htm;
+}
+location /RequestDenied {
+     return 412;
+}
+--- request
+PUT /foobar
+--- error_code: 405
```
Comment (wargio) [inline unit-tests/tests/33ignoreip.t:275]: which rule triggers?

File: unit-tests/tests/34ignorecidr.t [hunk 2]
```diff
@@ -247,3 +247,26 @@ GET /foobar?a=update/table
...
+     DeniedUrl "/RequestDenied";
+     root $TEST_NGINX_SERVROOT/html/;
+     index index.html index.htm;
+}
+location /RequestDenied {
+     return 412;
+}
+--- request
+PUT /foobar
+--- error_code: 405
```
Comment (wargio) [inline unit-tests/tests/34ignorecidr.t:272]: which rule triggers?

**Comment by wargio** [comment]
Let me merge this first: https://github.com/wargio/naxsi/pull/97 then please rebase.

File: unit-tests/tests/33ignoreip.t (hunk 1 above)
Comment (lubomudr) [reply unit-tests/tests/33ignoreip.t:275]: 11
PUT request without Content-type: header

**Comment by wargio** [comment]
done, you can rebase.

**Pull request merged**